.github/workflows/lock-versions.yml

```diff
@@ -6,7 +6,7 @@ on:
           description: 'List of branches to lock versions (ordered, comma separated)'
           required: true
           # 7.17 was intentionally skipped because it was added late and was bug fix only
-          default: '8.3,8.4,8.5,8.6,8.7,8.8,8.9,8.10,8.11'
+          default: '8.3,8.4,8.5,8.6,8.7,8.8,8.9,8.10,8.11,8.12'
```

Comment:
@terrancedejesus @eric-forte-elastic when are we going to start dropping versions?

Reply:
We discussed this during the training/exercise. Previously it was undecided when we were comfortable with dropping support to our agreed current-3 scope. Maybe we create a discussion and if all agrees, drop a certain list of minors.